src/App.jsx

```diff
@@ -14,57 +16,66 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setselectedGood] = useState('Jam');
```

Comment:
The function `setselectedGood` should be `setSelectedGood` to match the camelCase convention typically used in JavaScript and React for state setter functions. This will also align with the `useState` hook's return value.